apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -0,0 +1,1253 @@
+  height: 24px;
+  border-radius: 4px;
+  border: 1px solid var(--border);
+  background: var(--background);
+  font-size: 11px;
+  font-family: var(--font-mono, monospace);
+  padding: 0 4px;
+  width: 100%;
+  text-align: right;
+}
+
+:global(.cd-num:focus) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)